src/Components/Track/Track.js

```diff
@@ -0,0 +1,22 @@
+import './Track.css'
```

Comment:
This whole file is messy. Not sure what you want it to look like but clean up before PR